src/utils.py

```diff
@@ -86,15 +87,15 @@ def validate_request_data(input_json):
     :return: boolean, result
     """
     validate_string = "{} cannot be empty"
-    if 'git_url' not in input_json:
+    if 'git-url' not in input_json:
```

Comment:
@samuzzal-choudhury, the variable names used in payload are git_url, git_sha and email_ids.